src/steps/iNavigateTo.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This step will set a new hash to the app.
+ * The app should react to the navigation by providing proper routing.
+ *
+ * ToDo: Add [with <JSON>] to synopsis to pass data as json
+ *    regex: "(\\s+with\\s+(.+?))?"
+ *    parsing: var oObject = JSON.parse(sObject);
+ */
+module.exports = {
+    docs: {
+        description: "Performs navigation using hash.",
```

Comment:
- Do not include "." for consistency. 
- Also I would say "Navigates within the application using inner-app route". We may extends with Fiori Launchpad intent-based navigation later.